### PR TITLE
feat(dropdown-legacy): Adding component - TWIG-3

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-dropdown-legacy/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@ecl-twig/ec-component-dropdown-legacy",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.9.1",
```
Comment (emeryro) [inline src/ec/packages/ec-component-dropdown-legacy/package.json]: it should be version 2.11.0 (same for dependencies)

File: src/ec/packages/ec-component-dropdown-legacy/docs/dropdown-legacy.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated)
+- "extra_attributes" (optional) (array) (default: []) Extra attributes
+  - "name" (string) Attribute name, eg. 'data-test'
+  - "value" (string) Attribute value, eg: 'data-test-1'
+
+### Example:
+
+<!-- prettier-ignore -->
+```twig
+{% include 'path/to/expandable.html.twig' with {
```
[the remark this answers is not in the document]
Comment (papegaill) [reply src/ec/packages/ec-component-dropdown-legacy/docs/dropdown-legacy.md]: done

File: src/ec/packages/ec-component-dropdown-legacy/package.json (hunk 1 above)
Comment (papegaill) [reply src/ec/packages/ec-component-dropdown-legacy/package.json]: done

File: src/ec/packages/ec-component-dropdown-legacy/dropdown-legacy.html.twig [hunk 3]
```diff
@@ -0,0 +1,97 @@
+{% spaceless %}
+
+{#
+  Parameters:
+    - "id" (string) (default: '')
+    - "button" (predefined structure) :  Button component structure 
+    - "auto_init" (boolean) (default: true)
```
[the remark this answers is not in the document]
Comment (papegaill) [reply src/ec/packages/ec-component-dropdown-legacy/dropdown-legacy.html.twig]: I have removed the auto_init parameter as suggested. I also added the ECL autoinit globaly to preview-head.html file

**Comment by emeryro** [comment]
The component is good for me.
My only concern is about putting the auto-init globaly. If we do so, we would have to update existing component using auto-init to prevent double initialization.

**Comment by papegaill** [comment]
@emeryro Thanks you, yes all the components have been reviewed with this approach via the following PR https://github.com/ec-europa/ecl-twig/pull/152

**Pull request merged**